### DISCUSSION TITLE
Text Scale

### DISCUSSION THREAD

**Discussion by Ksylatarsh** [ikemen-engine/Ikemen-GO#1178] [Feature Request]
### Describe the bug

Text for some reason looks all mangled up and smaller than usual when using the Text SCTRL, I've checked everything (localcoord, scale, font, everything.) and I can't get it to display correctly. And yes I did change resolution of the game, to 348x196 which is the localcoord for everything (system, fight, character, etc), game resolution is doubled.

[State Text Win]
Type = Text
Trigger1 = !Time
Text = "Texting"
Font = F1
Align = 1
RemoveTime = -1
Pos = 174,96
Scale = 1,1
IgnoreHitPause = 1

My code ^

### To Reproduce

I can't really do much other than provide this test case, just start a match with the embedded kfm and it'll generate the broken text.
[test.zip](https://github.com/ikemen-engine/Ikemen-GO/files/11279696/test.zip)

### Expected behavior

Well, text should look correctly, much like example below.

### Screenshots / Video

Text generated using the Text SCTRL looks and using default fonts.
![image](https://user-images.githubusercontent.com/126763763/233254906-09b5c24d-3fe6-46e2-acd1-a30301087a8a.png)

Correct text generated in fight.def with same font.
![image](https://user-images.githubusercontent.com/126763763/233255146-06bbb55e-d9e3-4102-8353-15a5def646d8.png)

### Engine Version (or source code date)

Nightly - 04/19

### Operating system

Windows

**Comment by K4thos** [comment]
The size difference compared to lifebar is because you're using lifebar localcoord 348,196 and the text sctrl works with 240p coordinate space. 
Without changes to localcoord both lifebar and text have the same size:
![ikemen025](https://user-images.githubusercontent.com/8928195/233497857-7dec39b8-c264-4bbb-8074-1cdfab4e8bc4.png)

This state controller may be changed in future to work more like an explod, if someone puts extra work on it. There is also an open ticket suggesting to take character or self-defined localcoord into account: https://github.com/ikemen-engine/Ikemen-GO/discussions/1125
I don't consider the current behavior as a bug, although I agree that the implementation is not ideal.

**Reply by Ksylatarsh** [reply]
So, what you're saying is that the only solution is to stick with a multiple of 240 for resolution? Well that's a shame, thanks for answer.

**Comment by K4thos** [comment]
your test case showcases that the enforced 240p localcoord is a serious drawback that can't be counter-measured properly with scale (since it will lead to scaling artifacts due to unusual resolution), so I'm going to add localcoord support for this sctrl soon (via extra parameters).

**Comment by K4thos** [comment]
https://github.com/ikemen-engine/Ikemen-GO/pull/1179

**Comment by Ksylatarsh** [comment]
Thanks! :D
